doc/tested_devices/Eve Light Strip.md

```diff
@@ -0,0 +1,338 @@
+# Device Name: Eve Light Strip
+
+[Product Page](https://www.evehome.com/en/eve-light-strip) 
+
+
+# Step 1 Discovery
+Command - (venv) Christophers-MacBook-Air:HomekitPythonPlaying christopheryoung$ python -m homekit.discover >  pairingtest.txt
+
+## Output
+
+Name: Eve Light Strip 8606._hap._tcp.local.
+Url: http_impl://10.101.0.175:80
+Configuration number (c#): 1
+Feature Flags (ff): Supports HAP Pairing (Flag: 1)
+Device ID (id): 1A:EA:89:9D:04:AF
+Model Name (md): Eve Light Strip 20EAS9901
+Protocol Version (pv): 1.1
+State Number (s#): 1
+Status Flags (sf): Accessory has not been paired with any controllers. (Flag: 1)
+Category Identifier (ci): Lightbulb (Id: 5)
+
+# Step 2. unpaired identify
+
+(venv) Christophers-MacBook-Air:HomekitPythonPlaying christopheryoung$ python3 -m homekit.identify -d 1A:EA:89:9D:04:AF
+
+## Result:
+Eve Light strip flashes
+
+# Step 3. Initialize Controller Storage
+
+Created the file test_report.json
```

Comment:
This can be done with **init_controller_storage.py**.

Reply:
Sure. Do you want to add this to add this to the test documentation? Happy to make the modification and rerun the tests once we settle on what the process looks like.

Reply:
done, see https://github.com/jlusiardi/homekit_python/blob/add_device_reports/doc/HomeKit-Accessory-Testing-Guide.md#step-3---initialize-controller-storage-optional

Reply:
Got it. Also saw the rest of the testing steps were added. I’ll run through them and then post results to my fork for review before issuing a new PR.